### ISSUE TITLE
Share button not working

### ISSUE THREAD

**Issue by no-gravity** [albertxing/stool#9]
I get an error when using the share button.

Looks like it tries to post the code to some backend service.

Would be nice to have it just add the code to the #fragment of the url so it works independent of a backend service.

**Comment by albertxing** [comment]
GitHub deprecated anonymous Gist creation [earlier this year](https://blog.github.com/2018-02-18-deprecation-notice-removing-anonymous-gist-creation/), which is what stool used to store benchmarking code.

I _had_ considered storing encoded strings in the URL, but those tend to end up quite long and are not very wieldy 🤷‍♂️

**Comment by albertxing** [comment]
I will look into alternatives though, including using GitHub auth to share (of course, benchmarking would still be available without signing in)

**Comment by no-gravity** [comment]
Yes, the urls would be long. But they could be shortened via any url shortener.

**Comment by MarshallOfSound** [comment]
@albertxing The share button sounds pretty important, can we get this fixed?
